Resources/doc/PairStorage/pairstorage.rst

```diff
@@ -0,0 +1,79 @@
+PairStorage
+===========
+
+``PairStorage`` is internal service used to store run-time key-value parameters in ``ElasticSearch`` document for internal usage.
+
+Initial configuration
+---------------------
+
+Before using ``PairStorage``, Pair Document, along with all other ``ConnectionsBundle`` documents, must be registered in ``ElasticSearch`` manager, i.e.:
+
+.. code-block:: yaml
+
+  ongr_elasticsearch:
+      connections:
+          default:
+            ...
+      managers:
+          default:
+              connection: default
+              mappings:
+                  - ONGRConnectionsBundle
+..
+
+Usage
+-----
+
+``PairStorage`` is a simple service which stores key-value in ``ElasticSearch`` document.
+Where:
```

Comment:
You probably want an extra new line before this "Where", because it is displayed in the same paragraph as the previous line.